git-glossary.md

```diff
@@ -5,6 +5,13 @@ This git glossary contains definitions of terms used in this repository.
 There is another dcoument that describes a mental model of git that
 when understood can help one master git concepts at a deeper level. 
 
+## clone
+A clone of a [github](#github) repository is a local copy of a repository residing on [github](#github). git maintains
+a set of symbolc refernces to URL's for remote repositories. After the clone operation is complete, git sutomatically
```

Comment:
I'm confused, because i honestly don't know what symbolic references are, is it similar to symlinks? Something else? Can we just call it references? I guess the risk of calling it just references can be that people might mistake it for the references defined here in the glossary below

Reply:
Let me think about this. If you are confused by it, something needs to change. I am at Starbuck’s responding to these comments. I need to review this in an envorment more conducive to concentrating. 
BTW, I really appreciate your comments.

Reply:
It may be my lack of knowledge, i did a more careful search than earlier and found this:
https://git-scm.com/docs/gitglossary#gitglossary-aiddefsymrefasymref
Another approach (rather than changing what we call it) is to provide a link to a new word in our glossary
I'm happy we can work together, i always look forward to reading your messages because there's always something of substance for my brain to learn :) and also i think you/we have the potential to contribute something important to the community

Reply:
@SunyataZero I have an idea how to improve the git-info discussion of references. It will involve changes in a a variety of places. How would you feel about merging this pull request and I will create a new one that has all of the changes proposed in these comments? If you, agree please merge this pull request. If you are not inclined to do that, that is OK too. Please let me know what you think.